packages/malloy/src/lang/test/parse.spec.ts

```diff
@@ -1046,12 +1556,37 @@ describe('expressions', () => {
         x.type === 'number' &&
         x.e
       ) {
-        const firstFrag = x.e[0];
-        if (typeof firstFrag === 'string') {
-          expect(firstFrag).toContain('(');
-        } else {
-          fail('expression with parens compiled oddly');
-        }
+        expect(x).toMatchObject({
+          'e': [
+            {
+              'function': 'numberLiteral',
+              'literal': '1',
+              'type': 'dialect',
+            },
+            // TODO not sure why there are TWO sets of parentheses... A previous version of this test
+            // just checked that there were ANY parens, so that went under the radar. Not fixing now.
+            '+((',
+            {
+              'denominator': [
+                {
+                  'function': 'numberLiteral',
+                  'literal': '4',
+                  'type': 'dialect',
+                },
+              ],
+              'function': 'div',
+              'numerator': [
+                {
+                  'function': 'numberLiteral',
+                  'literal': '3',
+                  'type': 'dialect',
+                },
+              ],
+              'type': 'dialect',
+            },
+            '))',
+          ],
+        });
```

Comment:
This test broke when I made number literals be a dialect function, rather than just a string fragment. I re-wrote the test, which revealed that we're generating double parentheses and I'm not sure why.